_pages/en/strategy-target-solution-delivery-model.md

```diff
@@ -882,31 +871,41 @@ The people a development value stream uses are a set of DevOps teams.
 
 Organizing portfolios around development value streams enables visualizing the flow of work to produce solution, reduces handoffs and delays, allows faster learning and shorter time to market, supports leaner development and budgeting methods.
 
-**IT Solution**
+**IT Solution, Application, Software, and Data**
 
-An IT solution is a combination of one or more IT Products.
-It produces the environment within which end users operate.
+A standard definition is expected to be produced by EARB (see the [_Adopt, Build, Buy strategy’s coherent set of actions_](https://sara-sabr.github.io/ITStrategy/strategy-build-vs-buy.html#coherent-set-of-actions)). Until this is complete, the following definition and relationships are being used.
```

Comment:
Good way of setting the expectations, propose we need to update ABB accordingly as well.